dev/com.ibm.ws.beanvalidation.v20/bnd.bnd

```diff
@@ -18,14 +18,24 @@ Bundle-Description: Bean Validation Container v2.0; version=${bVersion}
 
 WS-TraceGroup: BeanValidation
 
-Export-Package: com.ibm.ws.beanvalidation.v20.config
+Import-Package: \
+  javax.validation.*;version="[2.0,3)",\
```

Comment:
this line can be removed, since we will import version 2.0 automatically since it's on the `-buildpath`

Reply:
Done.